integration_tests/k8s/k8sapi.go

```diff
@@ -118,7 +118,7 @@ func (c K8sClient) RestartPod(name string) error {
 	}
 	for pod := range pods {
 		if strings.Contains(pod, name) {
-			return c.ClientSet.CoreV1().Pods(Namespace).Delete(pod, &metav1.DeleteOptions{})
+			return c.ClientSet.CoreV1().Pods(Namespace).Delete(context.Background(), pod, metav1.DeleteOptions{})
```

Comment:
```suggestion
			return c.ClientSet.CoreV1().Pods(Namespace).Delete(context.TODO(), pod, metav1.DeleteOptions{})
```

Reply:
I see `TODO` context being used at multiple places. Is there any root context we can replace these with?